tests/test_1781_rdataframe_snapshot.py

```diff
@@ -49,7 +52,11 @@ def test_data_frame_vec_of_vec_of_real(tmp_path):
     )
     assert ak_array_in.to_list() == ak_array_out["x"].to_list()
 
-    with pytest.raises(SystemError):
+    # With `ROOT.ROOT.EnableImplicitMT(1)` a SystemError becomes a Warning:
```

Comment:
This is kind of strange that an exception becomes a warning. The warning is caught beneath, but it is empty...